.github/dependabot.yml

```
@@ -9,3 +9,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    groups:
+      all:
+        patterns:
+          - "*"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)